RENAME.md

```diff
@@ -0,0 +1,150 @@
+| org.grails                        | grails-plugin-datasource                        | org.apache.grails                               | grails-datasource                                             |                        |                        | grails-core            |
+| org.grails                        | grails-datastore-async                          | org.apache.grails.data                          | grails-datastore-async                                        |                        |                        | grails-data-mapping    |
+| org.grails                        | grails-datastore-core                           | org.apache.grails.data                          | grails-datastore-core                                         |                        |                        | grails-data-mapping    |
+| org.grails                        | grails-datastore-web                            | org.apache.grails.data                          | grails-datastore-web                                          |                        |                        | grails-data-mapping    |
+| org.grails                        | grails-dependencies                             | org.apache.grails                               | grails-dependencies                                           |                        |                        | grails-core            |
+| org.grails                        | grails-docs                                     | org.apache.grails                               | grails-docs-core                                              |                        |                        | grails-core            |
+| org.grails                        | grails-plugin-domain-class                      | org.apache.grails                               | grails-domain-class                                           |                        |                        | grails-core            |
+| org.grails                        | grails-encoder                                  | org.apache.grails.view                          | grails-encoder                                                |                        |                        | grails-core            |
+| org.grails.plugins                | events                                          | org.apache.grails                               | grails-events                                                 |                        |                        | grails-core            |
+| org.grails                        | grails-events                                   | org.apache.grails.event                         | grails-events-core                                            |                        |                        | grails-core            |
```

Comment:
I think using org.apache.grails.events is better, and the artifacts also have -events prefix.